src/Sarif/Sarif.csproj

```diff
@@ -96,5 +97,9 @@
 
   <Import Project="ToDotNet\ToDotNet.targets" />
 
-  <ProjectExtensions><VisualStudio><UserProperties CodeGenHints_1json__JSONSchema="http://json.schemastore.org/asmdef" /></VisualStudio></ProjectExtensions>
+  <ProjectExtensions>
+    <VisualStudio>
```

Comment:
>VisualStudio [](start = 5, length = 12)

This can be deleted from merge, I think.

Reply:
just removed! thank you

---
In reply to: [520763919](https://github.com/microsoft/sarif-sdk/pull/2148#discussion_r520763919) [](ancestors = 520763919)

Reply:
merge away!

---
In reply to: [520765577](https://github.com/microsoft/sarif-sdk/pull/2148#discussion_r520765577) [](ancestors = 520765577,520763919)